include/AL/alut.h

```diff
@@ -17,8 +17,10 @@ extern "C" {
 #endif
 
 #if defined(_WIN32) && !defined(_XBOX)
- #if defined (ALUT_BUILD_LIBRARY)
+ #if defined(ALUT_BUILD_LIBRARY)
   #define ALUT_API __declspec(dllexport)
+ #elif defined(ALUT_BUILD_STATIC)
```

Comment:
I don't see where this macro is set. Grep thru whole project for 'ALUT_BUILD_STATIC' only reveals this spot in the code.

Reply:
good catch